npm/test/clientfilemetataupload.test.ts

```diff
@@ -219,6 +219,30 @@ test("createMetadataInputsAndFileMap returns metadata inputs and file map", () =
   }
 })
 
+test("createMetadataInputsAndFileMap replaces double slash in metadata inputs file path with single slash", async () => {
+  const mockMetadata1WithDoubleSlash = <IFileMetadata>{
+    ...mockMetadata1,
+    path: "/path/to/file1"
+  }
+  const mockMetadata2WithDoubleSlash = <IFileMetadata>{
+    ...mockMetadata2,
+    path: "path2/to/file2"
```

Comment:
Neither of these examples look like they have a double slash in them, so it looks like test setup doesn't match the name of the test. Am I missing something?

Reply:
Ah, this was a mistake; I'll fix the naming. Thanks.